root/tag/TagIndex.js

```diff
@@ -65,28 +73,40 @@ function buildSection<T>(
   );
 }
 
-const TagIndex = (props: Props) => (
-  <TagLayout page="" tag={props.tag}>
-    {/*
+const TagIndex = (props: Props) => {
+  const genre = props.tag.genre;
+  return (
+    <TagLayout page="" tag={props.tag}>
+      {genre ? (
+        <>
+          <h2>{l('Genre')}</h2>
+          <p>
+            {exp.l('This tag is associated with the genre {genre}.',
+                   {genre: <EntityLink entity={genre} />})}
+          </p>
+        </>
+      ) : null}
+      {/*
       * The below use N_ln so languages with non-Germanic pluralization
       * rules (i.e., any that make number distinctions above the
       * threshold where we'll actually show the string) can translate
       * properly. However, the strings are the same in English because
       * we do not make a distinction other than for 1, which will never
       * show in this case.
       */}
-    {buildSection(props, 'area', l('Areas'), N_ln('See all {num} areas', 'See all {num} areas'))}
-    {buildSection(props, 'artist', l('Artists'), N_ln('See all {num} artists', 'See all {num} artists'))}
-    {buildSection(props, 'event', l('Events'), N_ln('See all {num} events', 'See all {num} events'))}
-    {buildSection(props, 'instrument', l('Instruments'), N_ln('See all {num} instruments', 'See all {num} instruments'))}
-    {buildSection(props, 'label', l('Labels'), N_ln('See all {num} labels', 'See all {num} labels'))}
-    {buildSection(props, 'place', l('Places'), N_ln('See all {num} places', 'See all {num} places'))}
-    {buildSection(props, 'release_group', l('Release Groups'), N_ln('See all {num} release groups', 'See all {num} release groups'))}
-    {buildSection(props, 'release', l('Releases'), N_ln('See all {num} releases', 'See all {num} releases'))}
-    {buildSection(props, 'recording', l('Recordings'), N_ln('See all {num} recordings', 'See all {num} recordings'))}
-    {buildSection(props, 'series', l('Series'), N_ln('See all {num} series', 'See all {num} series'))}
-    {buildSection(props, 'work', l('Works'), N_ln('See all {num} works', 'See all {num} works'))}
-  </TagLayout>
-);
+      {buildSection(props, 'area', l('Areas'), N_ln('See all {num} areas', 'See all {num} areas'))}
```

Comment:
Would be nice to commit large reindent apart, plus above comment is not correctly indented.

Reply:
But if I commit the reindent separately then I'll have a commit with the wrong style, isn't that bad?

Reply:
Oh right, that is just due to new block level, not fixing old indent.